### DISCUSSION TITLE
Adding a custom scope for topics

### DISCUSSION THREAD

**Discussion by jdesulme** [community/community#10917] [Other Feature Feedback, Questions, & Ideas]
We're leveraging topics to organize our code around teams within an EMU based organization. It would be helpful to add a scope based on topics.

**Comment by jdesulme** [comment]
Hey @colinwm @look any thoughts on this capabilty?

**Reply by look** [reply]
@jdesulme I like this idea but I can't promise we'll get to it soon. Thanks for the suggestion and stay tuned.

**Comment by raphberube** [comment]
Related to the topic custom scope, support for a topic qualifier in the search query would also be very useful.

**Comment by nielsbosma** [comment]
This would be very useful when searching for example created in a specific framework.

**Comment by funkswing** [comment]
Not being able to use `topic:` in GitHub code search greatly limits the ability to use Code Search effectively in our EMU organization. Like others have said we use topics within our Organization to organize our repos. Being able to filter our search to specific categories of repos using GitHub topics would greatly improve the day to day experience.
